test/fragment-generation-utils-test.js

```diff
@@ -831,6 +839,10 @@ describe('FragmentGenerationUtils', function() {
   });
 
   it('will halt generation after a certain time period', function() {
+    if(isDebug()) {
```

Comment:
Style guide requires a space: `if (isDebug()) {`

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

I think running `npm install` will automatically fix these things for you.

Reply:
`npm run fix` does the trick.

Reply:
thank you both!